server/form-pages/apply/check-your-answers/index.ts

```diff
@@ -9,7 +9,7 @@ import { Section, Task } from '../../utils/decorators'
   pages: [CheckYourAnswersPage],
 })
 @Section({
-  title: 'Check answers and submit',
+  title: 'Check answers',
```

Comment:
We could fixup/replace with the [earlier commit](https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/pull/378/commits/1b08657c870b58360d4697fd19521841ba8c240a) to this file.